bytecomp/translcore.ml

```diff
@@ -364,7 +378,8 @@ let prim_restore_raw_backtrace =
 
 let specialize_comparison table env ty =
   let (gencomp, intcomp, floatcomp, stringcomp, bytescomp,
-           nativeintcomp, int32comp, int64comp, _) = table in
+       nativeintcomp, int32comp, int64comp,
+       uint32comp, uint64comp, _) = table in
```

Comment:
(I would support having `nativeintcomp` on its own line and breaking a line before `_)` to get nice diffs in the future. There is no nit too small to be picked.)